provider/resources.go

```diff
@@ -708,10 +708,6 @@ func ProviderFromMeta(metaInfo *tfbridge.MetadataInfo) *tfbridge.ProviderInfo {
 					EnvVars: []string{"AWS_REGION", "AWS_DEFAULT_REGION"},
 				},
 			},
-			"skip_get_ec2_platforms": {
```

Comment:
I'm assuming that this was removed upstream, and so warned during the build => removed.

Reply:
This was removed on v5.0.0: https://github.com/hashicorp/terraform-provider-aws/releases/tag/v5.0.0
and no longer appears on https://registry.teraform.io/providers/hashicorp/aws/latest/docs